docs/local_tutorial.md

```diff
@@ -0,0 +1,234 @@
+# Solr on Kubernetes on local Mac
+
+This tutorial shows how to setup Solr under Kubernetes on your local mac. The plan is as follows:
+
+ 1. [Setup Docker for Mac with K8S](#setup-docker-for-mac-with-k8s)
+ 2. [Install an Ingress Controller to reach the cluster on localhost](#install-an-ingress-controller)
+ 3. [Install Solr Operator](#install-solr-operator)
+ 4. [Start your Solr cluster](#start-your-solr-cluster)
+ 5. [Create a collection and index some documents](#create-a-collection-and-index-some-documents)
+ 6. [Scale from 3 to 5 nodes](#scale-from-3-to-5-nodes)
+ 7. [Upgrade to newer Solr version](#upgrade-to-newer-version)
+ 8. [Install Kubernetes Dashboard (optional)](#install-kubernetes-dashboard-optional)
+ 9. [Delete the solrCloud cluster named 'example'](#delete-the-solrcloud-cluster-named-example)
+
+## Setup Docker for Mac with K8S
+
+```bash
+# Install Homebrew, if you don't have it already
+/bin/bash -c "$(curl -fsSL \
+	https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+
+# Install Docker Desktop for Mac (use edge version to get latest k8s)
+brew cask install docker-edge
+
+# Enable Kubernetes in Docker Settings, or run the command below:
+sed -i -e 's/"kubernetesEnabled" : false/"kubernetesEnabled" : true/g' \
+    ~/Library/Group\ Containers/group.com.docker/settings.json
+
+# Start Docker for mac from Finter, or run the command below
+open /Applications/Docker.app
+
+# Install Helm, which we'll use to install the operator, and 'watch'
+brew install helm watch
+```
+
+## Install an Ingress Controller
+
+Kubernetes services are by default only accessible from within the k8s cluster. To make them adressable from our laptop, we'll add an ingress controller
+
+```bash
+# Install the nginx ingress controller
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
```

Comment:
These urls are 404 now. I think they both can be replaced with this one instead https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud/deploy.yaml

Reply:
yep, checked their documentation and that looks right! Good catch